app/assets/stylesheets/_to-member-card.scss

```diff
@@ -1,84 +1,170 @@
-.to-member-card {
+.to-member-card, .panel-default, .simple_form {
   & {
```

Comment:
No need to use `&` here (omitting the "&" refers to the root symbol by default).

Reply:
The coding style of putting "& {" in some sccs files was already in the code, the 2nd line was not modified. The string "& {"  could be found 8 more times in the sccs files. I'll remove them.